six/common/builtins/aggregator.c

```diff
@@ -275,11 +276,28 @@ static PyObject *submit_event(PyObject *self, PyObject *args)
     // send the event
     cb_submit_event(check_id, ev);
 
-    // cleanup and return
-    if (py_tags != NULL) {
+    //Success
+    Py_INCREF(Py_None);
+    retval = Py_None;
+
+tags_cleanup:
```

Comment:
doesn't look we're using this `tags_cleanup` label, you can probably drop it.

Reply:
I ninja'd you at this one :)